tests/fortran/client_test_ensemble.F90

```diff
@@ -142,5 +142,22 @@ program main
   error stop
 endif
 
+result = client%delete_model(model_key)
+if (result .ne. SRNoError) stop
```

Comment:
These should be `error stop` otherwise these errors won't register as a failed test. There's a bigger chore to change that elsewhere within the Fortran tests, but for now would you mind updating them in `client_test_ensemble.F90` for now?